syndesis/src/app/App.tsx

```diff
@@ -64,36 +65,38 @@ export class App extends React.Component {
   }
 
   public renderNavbar() {
+    const { t } = this.props;
```

Comment:
Can't we use the `NamespacesConsumer` render prop here instead?

Reply:
This method returns an array. The only way I think we can use `NamespacesConsumer` would be to wrap **each** of the 5 `PfVerticalNavItem`s and I thought that would make the code fairly ugly. Do you want me to go ahead and do that?

Reply:
You should be able to change that array to use a [Fragment](https://reactjs.org/docs/fragments.html) instead (dunno why I didn't do that at the time).

```jsx
public renderNavbar() {
    return (
      <>
        <PfVerticalNavItem
          icon={'fa fa-tachometer'}
          to={'/'}
          exact={true}
          label={'Home'}
          key={1}
        />
        <PfVerticalNavItem
          icon={'pficon pficon-integration'}
          to={'/integrations'}
          label={'Integrations'}
          key={2}
        />
        <PfVerticalNavItem
          icon={'pficon pficon-plugged'}
          to={'/connections'}
          label={'Connections'}
          key={3}
        />
        <PfVerticalNavItem
          icon={'fa fa-cube'}
          to={'/customizations'}
          label={'Customizations'}
          key={4}
        />
        <PfVerticalNavItem
          icon={'pficon pficon-settings'}
          to={'/settings'}
          label={'Settings'}
          key={5}
        />
      </>
    );
```

Reply:
Yes that works. So converted over to using NamespacesConsumer. Much better!